Stop should cancel "completed" callbacks (#15)
Fraktality/spr: README.md

Stops animations for a particular property.

> If a property is not specified, all properties belonging to the instance will stop animating.
> 
> Additionally, this also cancels any 'completed' callbacks associated
> 

Reviewer: ```suggestion
If a property is not specified, all properties belonging to the instance will stop animating.
Any completed callbacks associated with the instance will be canceled.

```
